docs/assets/highlight.css

```diff
@@ -0,0 +1,22 @@
+:root {
```

Comment:
What are all of these assets used for?

Reply:
These are generated docs - they aren't actually made by me, they are made by Typedoc, and are automatically generated by a github workflow (and committed to the repo!) once meaningful API changes are introduced. 

I have just now enabled GHpages for this repo, so as soon as I make another commit / it gets around to building, these docs pages will be available at https://thegnarco.github.io/gnar-cli/

(We'd normally want to set this to the `main` branch, but since these docs don't exist there yet, I set up pages to look here.)

Reply:
( I will need to change the settings ab it, as these docs are only minorly helpful and focused on exports vs. general classes, but it works!)

Reply:
[bb0febf](https://github.com/TheGnarCo/gnar-cli/pull/29/commits/bb0febfecaea9264b85e225d32d03d7bc637b148) Is an example of an auto-generated doc update PR I didn't run anything locally, but this was created once the conditions for the typedoc changed. Docs are [here](https://thegnarco.github.io/gnar-cli/)!